core/templates/pages/library-page/search-bar/search-bar.component.css

```diff
@@ -277,19 +298,52 @@ oppia-search-bar .oppia-input-group {
   }
 }
 
+/* @media (max-width: 1600px) {
+  oppia-search-bar .oppia-input-group {
+    width: 52.5em;
+  }
+} */
+
```

Comment:
please only assign reviewers, when you review your pr first by yourself.

Reply:
Done PTAL